day1/spec/day1_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "day1"
+
+describe Day1 do
+  before(:each) do
+    @day1 = Day1.from_file("spec/test_input.txt")
```

Comment:
You tend to want to isolate full file specs (people would classify them as either functional or integration depending on the camp) from unit specs. You only really need one test hitting this data, then everything else can use a subject that's easier to trace like `new Day1([...])`

Reply:
Nice, I definitely see what you were saying. I like that this approach will also show how the simple instantiation is warranted. Just pushed up some spec refactoring as suggested in https://github.com/coopergillan/advent-of-ruby-2020/commit/753995d486e481dcff106e1f6cdae7b5cf06d65c.